--- isa-l-2.32.0/configure.ac.orig	2026-03-04 14:56:50.000000000 +0100
+++ isa-l-2.32.0/configure.ac	2026-05-14 21:09:37.443435923 +0200
@@ -23,6 +23,10 @@ AM_PROG_AS
 
 AC_CANONICAL_HOST
 CPU=""
+
+AS_CASE([$host],
+  [x86_64-*-gnux32], [],
+  [
 AS_CASE([$host_cpu],
   [x86_64], [CPU="x86_64"],
   [amd64], [CPU="x86_64"],
@@ -32,6 +36,7 @@ AS_CASE([$host_cpu],
   [ppc64le], [CPU="ppc64le"],
   [riscv64], [CPU="riscv64"],
 )
+])
 AM_CONDITIONAL([CPU_X86_64], [test "$CPU" = "x86_64"])
 AM_CONDITIONAL([CPU_AARCH64], [test "$CPU" = "aarch64"])
 AM_CONDITIONAL([CPU_PPC64LE], [test "$CPU" = "ppc64le"])
