--- isa-l-2.30.0/Makefile.am~	2020-11-07 02:08:16.000000000 +0100
+++ isa-l-2.30.0/Makefile.am	2022-09-04 13:32:49.888076730 +0200
@@ -58,16 +58,6 @@
 pkginclude_HEADERS += $(sort ${extern_hdrs})
 libisal_la_SOURCES = ${lsrc}
 
-if CPU_X86_64
-libisal_la_SOURCES += ${lsrc_x86_64}
-other_tests += ${other_tests_x86_64}
-endif
-
-if CPU_X86_32
-libisal_la_SOURCES += ${lsrc_x86_32}
-other_tests += ${other_tests_x86_32}
-endif
-
 if CPU_AARCH64
 libisal_la_SOURCES += ${lsrc_aarch64}
 other_tests += ${other_tests_aarch64}
@@ -78,9 +68,7 @@
 other_tests += ${other_tests_ppc64le}
 endif
 
-if CPU_UNDEFINED
 libisal_la_SOURCES += ${lsrc_base_aliases}
-endif
 
 nobase_include_HEADERS = isa-l.h
 libisal_la_LDFLAGS = $(AM_LDFLAGS) \
